nltk/translate/bleu_score.py

```diff
@@ -58,8 +58,8 @@ def sentence_bleu(references, hypothesis, weights=(0.25, 0.25, 0.25, 0.25),
     >>> sentence_bleu([reference1, reference2, reference3], hypothesis1) # doctest: +ELLIPSIS
     0.5045...
 
-    >>> sentence_bleu([reference1, reference2, reference3], hypothesis2) # doctest: +ELLIPSIS
-    0.3969...
+    >>> round(sentence_bleu([reference1, reference2, reference3], hypothesis2),4) # doctest: +ELLIPSIS
+    0.0
```

Comment:
Is the change in the value justified? The previous value is taken from the paper (if I remember correctly).

If it's really 0 now, the reason should be given.

Reply:
I know added a more in-depth description here why 0 is the expected value here.

The value was not taken from the paper. The thing that was taken from the paper were the reference and hypothesis examples. But not the values.